scripts/check-diff.sh

```diff
@@ -585,6 +585,10 @@ function install_wp {
 	mkdir -p "$WP_CORE_DIR/src/wp-includes/css/dist/block-library"
 	touch "$WP_CORE_DIR/src/wp-includes/css/dist/block-library/style.css"
 
+	# Download required asset dependency files
+	local SVN_CORE_URL=${SVN_URL/develop/core}
```

Comment:
Could we use the full `wp-includes` directory from the release repository?

https://core.svn.wordpress.org/trunk/wp-includes

That would solve both the `block-library` issue above and this one.

Reply:
Sure that makes sense.